src/Sarif.FunctionalTests/v2/ConverterTestData/PREfast/Expected-050.xml

```diff
@@ -1,100 +1,279 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<DEFECTS><DEFECT><SFA><FILEPATH>c:\somepath\</FILEPATH><FILENAME>test050.cpp</FILENAME><LINE>7</LINE><COLUMN>12</COLUMN></SFA><DEFECTCODE>26011</DEFECTCODE><DESCRIPTION>Potential read underflow using expression 'a[i]'
-PREFAST_NEWLINE
-Buffer accessed is a
-PREFAST_NEWLINE
-Buffer is of length 10 elements (4 bytes/element) [size of variable]
-PREFAST_NEWLINE
-Accessing the element (4 bytes/element) at element offset i`4
-PREFAST_NEWLINE
+<DEFECTS>
```

Comment:
>DEFECTS [](start = 1, length = 7)

Here, I've expressly expanded some PREfast log file content based on reformatting in VS. This has slightly increased the variability of the test content.